sdk/spring/spring-cloud-azure-actuator/src/main/java/com/azure/spring/cloud/actuator/cosmos/CosmosHealthIndicator.java

```diff
@@ -45,23 +46,31 @@ public CosmosHealthIndicator(CosmosAsyncClient cosmosAsyncClient, String databas
 
     @Override
     protected void doHealthCheck(Builder builder) {
+        if (database == null) {
+            builder.status(Status.UNKNOWN).withDetail("Database not configured",
+                "The option of `spring.cloud.azure.cosmos.database` is not configured!");
+            return;
+        }
+
         CosmosDatabaseResponse response = this.cosmosAsyncClient.getDatabase(database)
-            .read()
-            .block(timeout);
+                                                                .read()
+                                                                .block(timeout);
 
         if (response != null) {
             LOGGER.info("The health indicator cost {} RUs, cosmos uri: {}, dbName: {}",
                 response.getRequestCharge(), endpoint, database);
-        }
-        if (response == null) {
-            builder.down();
+            builder.up()
+                   .withDetail("RUs", response.getRequestCharge())
```

Comment:
Let's remove RUs for now, and we can add it if there's a customer ask.

Reply:
OK